docs/distributions/fedora/faq.md

```diff
@@ -5,3 +5,23 @@ If you already have Bootcamp installed, you might notice that the boot option fo
 # My boot hangs before getting to the installer
 
 This may be due to differences between USB-C to USB-A adapters. Try a different one if it is not working.
+
+# My touchbar is blank
```

Comment:
In the postinstall guide, there is a dedicated section for the touchbar where such fixes are there.